e2e/tests/functional/plugins/notebook/notebook.e2e.spec.js

```diff
@@ -32,9 +32,19 @@ const path = require('path');
 const NOTEBOOK_NAME = 'Notebook';
 
 test.describe('Notebook CRUD Operations', () => {
-  test.fixme('Can create a Notebook Object', async ({ page }) => {
-    //Create domain object
+  test('Can create a Notebook Object', async ({ page }) => {
```

Comment:
I think most of this test case (Creating the Notebook object and navigating to it) can actually be encompassed completely within a `test.beforeEach()` hook. See our [e2e example template](https://github.com/nasa/openmct/blob/d63df98285ac7774574684adcb566bfcd7705154/e2e/tests/framework/exampleTemplate.e2e.spec.js#L66) for a good example of this.

Reply:
Hi @ozyx thanks for the helpful feedback. I looked at the e2e example template. Am I right in thinking the I put the `test.beforeEach()` hook  at the `test.fixme`?
```
let Notebook;
test.beforeEach('Can create a Notebook Object', async ({ page})  => {} 
```

Reply:
Correct, the `test.beforeEach()` should be defined within the test suite (`test.describe()`).